dipy/reconst/csdeconv.py

```diff
@@ -463,17 +466,17 @@ def estimate_response(gtab, evals, S0):
     return single_tensor(gtab, S0, evals, evecs, snr=None)
 
 
-def forward_sdt_deconv_mat(ratio, n, r2_term=False):
+def forward_sdt_deconv_mat(ratio, l_values, r2_term=False):
```

Comment:
here, it is missing `@deprecated_params('n', 'l_values', since='1.9', until='2.0')